src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -37,16 +37,21 @@ interface BuildLocator {
      * By default only successful builds are returned, call this method to include failed builds as well.
      */
     fun withAnyStatus() : BuildLocator
-
     fun withStatus(status: BuildStatus): BuildLocator
+
     fun withTag(tag: String): BuildLocator
 
-    fun withBranch(branch: String): BuildLocator
+    /**
+     * By default failed to start builds are filtered out, call this method to include them.
+     */
+    fun withAnyFailedToStart(): BuildLocator
+    fun withFailedToStart(value: Boolean): BuildLocator
```

Comment:
Why do you need to have method to revert options? It would complicate API and may be useful only if someone wants to pass BuildLocator instance between methods. Also if we add it behavior of BuildLocator will become dependent on order in which the methods are called, it may cause problems. Do you have any real use case there it's necessary?